man/gemfile.5.ronn

```diff
@@ -20,18 +20,19 @@ a number of methods used to describe the gem requirements.
 At the top of the `Gemfile`, add a line for the `Rubygems` source that contains
 the gems listed in the `Gemfile`.
 
-    source "https://rubygems.org"
-
-It is possible, but not recommended as of Bundler 1.7, to add multiple global
-`source` lines. Each of these `source`s `MUST` be a valid Rubygems repository.
-
-Sources are checked for gems following the heuristics described in
-[SOURCE PRIORITY][]. If a gem is found in more than one global source, Bundler
-will print a warning after installing the gem indicating which source was used,
-and listing the other sources where the gem is available. A specific source can
-be selected for gems that need to use a non-standard repository, suppressing
-this warning, by using the [`:source` option](#SOURCE-source-) or a
-[`source` block](#BLOCK-FORM-OF-SOURCE-GIT-PATH-GROUP-and-PLATFORMS).
+Global source lines are a security risk and should not be used as they can lead
```

Comment:
This doesn't mention single-gem `:source` options.
I think the audience the docs should be written for is not just for new people writing Gemfile from scratch, but people already having Gemfiles and trying to understand how they actually behave.

Worse, other places in the file do describe `:source` as something that sounds safe, eg.:

> This forces the gem to be loaded from this source and ignores any global sources

but [the blog post][1] specifically shows that:

```
source "http://public.org"
gem "foo"
gem "bar", source: "http://private.com"
gem "baz"
```
is unsafe.  I suppose all places mentioning `source` / `:source` should be touched.

Are `:source` exceptions safe if added inside a `source 'https://rubygems.org' do` block?  Are they always bad?

[1]: http://collectiveidea.com/blog/archives/2016/10/06/bundlers-multiple-source-security-vulnerability/